packages/grid/src/styled/pane/StyledPaneSplitterButton.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
+import { rgba } from 'polished';
+import { ISplitterButtonProps, Orientation } from '../../types';
+import { IconButton } from '@zendeskgarden/react-buttons';
+import { StyledPaneSplitter } from './StyledPaneSplitter';
+
+const COMPONENT_ID = 'pane.splitter_button';
+
+interface IStyledSplitterButtonProps extends ISplitterButtonProps {
+  orientation: Orientation;
+}
+
+const colorStyles = (props: ThemeProps<DefaultTheme>) => {
+  const backgroundColor = getColor('background', 0, props.theme);
+  const hoverColor = getColor('primaryHue', 100, props.theme);
+  const focusRingColor = getColor('primaryHue', 300, props.theme);
+  const activeColor = getColor('primaryHue', 300, props.theme);
+  const boxShadow = props.theme.shadows.lg('4px', '8px', rgba(4, 68, 77, 0.15));
+  const focusRingShadow = props.theme.shadows.md(focusRingColor!);
+
+  return css`
+    box-shadow: ${boxShadow};
+    background-color: ${backgroundColor};
+
+    &:hover {
+      background-color: ${hoverColor};
+    }
+
+    &[data-garden-focus-visible] {
+      box-shadow: ${boxShadow}, ${focusRingShadow};
+    }
+
+    &:active {
+      background-color: ${activeColor};
+    }
+  `;
+};
```

Comment:
The `ChevronButton`/`IconButton` already has these styles – shouldn't need to repeat here.